data/remnant missions.txt

```diff
@@ -655,149 +655,288 @@ mission "Remnant: Learn Sign Follow Up"
 
 
 
+mission "Remnant: Tech Retrieval"
+	name "Remnant Human Tech Retrieval"
+	description "The Remnant are interested in learning what advances humanity has made over the past centuries. Bring them samples of advanced technology."
+	source "Viminal"
+	to offer
+		has "Remnant: Technology Available: offered"
+		random < 30
+	on offer
+		"remnant taely" ++
+		conversation
+			`As you enter the spaceport, you notice someone across the plaza appears to have been waiting for you. A tall, solidly built woman in short-sleeved coveralls stands up from where she was sitting near the information desk. With a few quick gestures at the person behind the counter, she heads towards you with a confident stride. As she approaches you note that the crowds of people moving around the plaza seem to staying out of her way as if she were someone important. In contrast, the tools at her belt and the wear marks on her clothing suggest someone who is more comfortable inside a ship access duct than behind a desk.`
```

Comment:
Comma after `As she approaches`
```suggestion
			`As you enter the spaceport, you notice someone across the plaza appears to have been waiting for you. A tall, solidly built woman in short-sleeved coveralls stands up from where she was sitting near the information desk. With a few quick gestures at the person behind the counter, she heads towards you with a confident stride. As she approaches, you note that the crowds of people moving around the plaza seem to staying out of her way as if she were someone important. In contrast, the tools at her belt and the wear marks on her clothing suggest someone who is more comfortable inside a ship access duct than behind a desk.`
```
Maybe change this last sentence somehow? It's implying that someone of importance would be behind a desk, so it's surprising to the player that people are staying out of the way of someone who looks like a factory worker. Maybe use that characterization of factory worker somehow; "she looks like someone who'd work in a Syndicate factory, not anyone of importance," or something like that.

Reply:
I was going more for a ship's engineering grunt or low ranking mechanic rather than factory worker.

Changed the latter part of that to:
```As she approaches, you note that the crowds of people moving around the plaza seem to staying out of her way as if she were someone important. In contrast, she reminds you of the mechanics who maintained the textile factories back on New Boston. In particular, the tools at her belt and the wear marks on her clothing suggest someone who is more comfortable inside a ship access duct than behind a desk.```